src/styles/_catalog.scss

```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: 300ms;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.